src/ontology/mondo-ingest.Makefile

```diff
@@ -433,6 +450,17 @@ $(COMPONENTSDIR)/%.db: $(COMPONENTSDIR)/%.owl
 	@rm -f .template.db.tmp
 	@rm -f $(COMPONENTSDIR)/$*-relation-graph.tsv.gz
 
+# todo: Temporarily need this goal until issue where where `Makefile` `$(COMPONENTSDIR)/%` goal overriding `$(COMPONENTSDIR)/%.db` is fixed. Discussion: https://github.com/monarch-initiative/mondo-ingest/pull/299/files#r1198555270
+.PHONY: %-db
+%-db: $(COMPONENTSDIR)/%.owl
```

Comment:
Please make an issue about this and assign it to yourself.

Reply:
Will do! and will take care of when https://github.com/INCATools/ontology-development-kit/pull/880 is in `odk:dev`.